### DISCUSSION TITLE
How do I determine versions of Quarto dependencies?

### DISCUSSION THREAD

**Discussion by kbvernon** [quarto-dev/quarto-cli#10319] [Q&A] [answered]
### Description

I am aware of `quarto check`, but that doesn't give version information for _everything_ that Quarto uses. So, how would I, for example, find the version of Reveal.js used in the most recent release of Quarto? 

This is a follow-up to this question: https://github.com/quarto-dev/quarto-cli/discussions/6873.

Thanks!

**Comment by cscheid** [comment]
_Everything_ is, plainly speaking, an impractical request. Does that mean every library that we get, directly or indirectly, from NPM? From deno.land? Dependencies that we get indirectly from ObservableHQ's runtime?

(We should report more. But "everything" is not feasible.)

**Reply by kbvernon** [reply]
Ha. Sorry. I worded that poorly. Following the question in 6873, I'm mainly interested in what the current version of Reveal.js is, but I thought maybe there was a more general way of finding versions being used.

**Reply by cscheid** [reply, marked as answer]
https://github.com/quarto-dev/quarto-cli/blob/main/configuration#L39 currently points to the commit from the revealjs repo that we're using. When we're using a release, that will be there as well.

**Reply by cderv** [reply]
@cscheid - Idea: Retrieve some information in this file automatically when building quarto-web by querying the repo and show those information somewhere in the website ? 

Otherwise, simpler version: Add this information as a FAQ question and point to the configuration file. Quite easy, so I may do this anyway !

**Reply by mcanouil** [reply]
I was pretty sure there was a page with partial information on dependencies but cannot find it anymore at least via web search.

Edit: I found it. It's the license page which contains some version, mostly for Bootstrap/Bootswatch: https://quarto.org/license.html (the version is not correct by the way)

**Reply by cderv** [reply]
I forgot this page - it is for Licence mainly, and we should probably remove the version number from as we don't have a good way to update it easily, and other don't have version
